### ISSUE TITLE
The commands for Civil-Comments

### ISSUE THREAD

**Issue by ghzhang233** [anniesch/jtt#5]
Hi. Thank you for your brilliant work, and I really like it, but I've met some problems during reproducing the results.

I noticed that the commands for Civil-Comments in the README are not consistent with those reported in the paper. For example, for the initial model, the learning rate is 1e-5 while it's 2e-5 in README. Batch size and Num Epoch are also different. Besides, it was said in another issue that _civil_comments_analysis.py_  should be used for evaluation rather than the _analysis.py_ as stated in README.

Could you please provide the commands with the exact same hyper-parameter settings as reported in the paper? I really appreciate your help! Thank you so much :)
